cmd/dmsg-discovery/internal/api/api.go

```diff
@@ -24,8 +27,24 @@ const maxGetAvailableServersResult = 512
 // API represents the api of the dmsg-discovery service`
 type API struct {
 	http.Handler
-	db       store.Storer
-	testMode bool
+	db              store.Storer
+	testMode        bool
+	startedAt       time.Time
+	sMu             sync.Mutex
+	numberOfClients int64
+	numberOfServers int64
+	error           string
+}
+
+// HealthCheckResponse is struct of /health endpoint
+type HealthCheckResponse struct {
+	BuildInfo            *buildinfo.Info `json:"build_info"`
+	NumberOfClients      int64           `json:"clients"`
+	NumberOfServers      int64           `json:"servers"`
+	StartedAt            time.Time       `json:"started_at,omitempty"`
+	AvgPackagesPerMinute uint64          `json:"average_packages_per_minute"`
```

Comment:
`dmsg.Discovery` does not handle any dmsg packets itself. This only should be returned for `dmsg.Servers`